### PR TITLE
Bug 1115247 - Create ViewManager to replace render.js

### PR DESCRIPTION
**Pull request by RudyLu**

### CODE REVIEW

File: apps/keyboard/js/keyboard/view_manager.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+
+    if (flags.showCandidatePanel) {
+      this.showCandidates([]);
+    }
+  }
+
+  // Make sure the container is switched to the current uppercase state.
+  pageView.setUpperCaseLock({
+    isUpperCase: flags.uppercase,
+  isUpperCaseLocked: false
```
Comment (timdream) [inline apps/keyboard/js/keyboard/view_manager.js]: Indent.

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5e08351f81d1)

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9eae984e90f9)

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dd375257778c)

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f9e3fd461bff)

**Pull request merged**